docs/topics/forms/index.txt

```diff
@@ -536,7 +536,7 @@ Then you can configure the :setting:`FORM_RENDERER` setting:
     from django.forms.renderers import TemplatesSetting
 
 
-    class CustomFormRenderer(TemplatesSetting):
+    class CustomFormRenderer(DjangoTemplates):
```

Comment:
Should the imports here and below also be updated? I don't think `DjangoTemplates` is in scope here.

Reply:
Of course, good catch thank you